src/setup/demo.md

```diff
@@ -0,0 +1,227 @@
+  --test-opener=assets/dataset/opener.py \
+  --metrics=assets/objective/ \
+  --fake-data-samples
+```
+
+### Training & Testing tasks
+
+```sh
+# Train
+python3 assets/algo/algo.py train \
+  --debug \
+  --opener-path assets/dataset/opener.py \
+  --data-samples-path assets/train_data_samples \
+  --output-model-path assets/model/model \
+  --log-path assets/logs/train.log
+
+# Predict
+python assets/algo/algo.py predict \
+  --debug \
+  --opener-path assets/dataset/opener.py \
+  --data-samples-path assets/train_data_samples \
+  --output-predictions-path assets/pred-train.csv \
+  --models-path assets/model/ \
+  --log-path assets/logs/train_predict.log \
+  model
+
+# Performance
+python assets/objective/metrics.py \
+  --debug \
+  --opener-path assets/dataset/opener.py \
+  --data-samples-path assets/train_data_samples \
+  --input-predictions-path assets/pred-train.csv \
+  --output-perf-path assets/perf-train.json \
+  --log-path assets/logs/train_metrics.log
+
+# Test / predict
+python assets/algo/algo.py predict \
+  --debug \
+  --opener-path assets/dataset/opener.py \
+  --data-samples-path assets/test_data_samples \
+  --output-predictions-path assets/pred-test.csv \
+  --models-path assets/model/ \
+  --log-path assets/logs/test_predict.log \
+  model
+
+# Test / performance
+python assets/objective/metrics.py \
+  --debug \
+  --opener-path assets/dataset/opener.py \
+  --data-samples-path assets/test_data_samples \
+  --input-predictions-path assets/pred-test.csv \
+  --output-perf-path assets/perf-test.json \
+  --log-path assets/logs/test_metrics.log
+```
+
+You will need to update or follow the relevant path to add your assets or get the logs and model.
+
+## 3. Resources
+
+### Find help
+
+Feeling lost? Remember, you can use `substra --help` anytime:
+
+```sh
+Usage: substra [OPTIONS] COMMAND [ARGS]...
+
+  Substra Command Line Interface.
+
+  For help using this tool, please open an issue on the Github
+  repository: https://github.com/SubstraFoundation/substra
+
+Options:
+  --version  Show the version and exit.
+  --help     Show this message and exit.
+
+Commands:
+  add          Add new asset to Substra platform.
+  cancel       Cancel execution of an asset.
+  config       Add profile to config file.
+  describe     Display asset description.
+  download     Download asset implementation.
+  get          Get asset definition.
+  leaderboard  Display objective leaderboard
+  list         List assets.
+  login        Login to the Substra platform.
+  run-local    Run local.
+  update       Update asset.
+```
+
+You can also get help for specific commands, for example `substra download --help`.
+
+### Get inspired
+
+The [Titanic example](https://github.com/SubstraFoundation/substra/blob/master/examples/titanic/README.md), included in the Substra repository, is ready to run but you can also have a look at the [community examples](https://github.com/SubstraFoundation/substra-examples):
+
```

Comment:
should we do a part about the CLI?

Reply:
What do you mean?
There are examples of CLI usage in this page, in the titanic example page, a (short) section about `substra --help` just before this one, and the link to the CLI documentation just after this section.
Do you think about something in particular that is missing or not clear?

Reply:
Explain how to connect and what you can see (or link to the CLI documentation page)